src/backend/db/videos.js

```diff
@@ -6,16 +6,91 @@
 export const videos = [
   {
     _id: "Wo5dMEP_BbI",
-    title: "Awesome Video about Coding",
+    title: "Kevin Hart - I'm Scared Of Ostriches",
     description:
-      "At vero eos et accusamus et iusto odio dignissimos ducimus qui blanditiis praesentium voluptatum deleniti atque corrupti quos dolores et quas molestias excepturi sint occaecati cupiditate non provident, similique sunt in culpa qui officia deserunt mollitia animi, id est laborum et dolorum fuga.",
-    creator: "Soham Shah",
+      "Comedy superstart #KevinHart (Think Like A Man, Ride Along, Top Five) graced the Montreal stage at the Just for Laughs ...",
+    creator: "Kevin Hart",
+    channelName: "Just for Laughs",
+    img: "https://i.ytimg.com/vi/WmIS_icNcLk/hq720.jpg?sqp=-oaymwEcCOgCEMoBSFXyq4qpAw4IARUAAIhCGAFwAcABBg==&rs=AOn4CLBKf25HoKWqAD4Wx6GuqOWMOWTuDA",
+    year: 7,
+    videoViews: "8.2M",
   },
   {
-    _id: "F_Riqjdh2oM",
-    title: "Neural Networks from Scratch - P.1 Intro and Neuron Code",
+    _id: "Adffdgssj_BbK",
+    title: "Rowan Atkinson - Going On Your First Date",
+    description:
+      "RowanAtkinson, actor and comedian best known for his work as #MrBean , hilariously acts out what a #firstdate is really like.",
+    creator: "Rowan Atkinson",
+    channelName: "Just for Laughs",
+    img: "https://i.ytimg.com/vi/mm1HgQbXFEE/hq720.jpg?sqp=-oaymwEcCOgCEMoBSFXyq4qpAw4IARUAAIhCGAFwAcABBg==&rs=AOn4CLCA804r1Yv3ItqhsZsEBzUmemDadA",
+    year: 6,
+    videoViews: "7.4M",
+  },
+  {
+    _id: "Y_RddhshYRJ",
+    title: "Dave Chappelle - Super Heroes Are Bad Role Models",
     creator: "Sentdex",
     description:
-      "At vero eos et accusamus et iusto odio dignissimos ducimus qui blanditiis praesentium voluptatum deleniti atque corrupti quos dolores et quas molestias excepturi sint occaecati cupiditate non provident, similique sunt in culpa qui officia deserunt mollitia animi, id est laborum et dolorum fuga.",
+      "DaveChappelle (Chappelle's Show, Half Baked… you know who he is!) on #Montreal 's black community, racial tension, and his ...",
+    creator: "Dave Chappelle",
+    channelName: "Just for Laughs",
+    img: "https://i.ytimg.com/vi/zNyqYOUxBrY/hq720.jpg?sqp=-oaymwEcCOgCEMoBSFXyq4qpAw4IARUAAIhCGAFwAcABBg==&rs=AOn4CLBnAOZKEEmxm0NUkHjwCgDhUqWl4A",
+    year: 6,
+    videoViews: "7M",
```

Comment:
suggestion: this key (videoViews) can be just views, no need to be explicit for this alone, rest of the keys belongs to video only

Reply:
Yahh make sense, I will make change that